build/package/Installer.MSI.targets

```diff
@@ -174,7 +174,7 @@
 
     <Target Name="GenerateSdkMSBuildExtensionsNupkg"
             DependsOnTargets="Init;Layout;MsiTargetsSetupInputOutputs;GenerateSdkBundle"
-            Condition=" '$(OS)' == 'Windows_NT'"
+            Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' "
```

Comment:
Why was this condition added here?

Reply:
@nguerrera 
The condition is present for the publish steps; see: https://github.com/dotnet/cli/blob/bcc48613fb2b7696576529cd8219f07c0dcbac73/build/publish/PublishNupkgToBlobFeed.targets#L11